### DISCUSSION TITLE
[Q&A] different Inventory Frequency for a few clients

### DISCUSSION THREAD

**Discussion by AndiMb** [glpi-project/glpi-agent#468] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Is there a way to change the inventory frequency of the GLPI agent for a few clients different from the server value of 24 h? I know, that I can use scheduled tasks running inventory and injectory, but it would be easier to use the agent.

**Comment by g-bougard** [comment]
Hi @AndiMb 

this is a global value set on server-side.

The only way I can this of (skipping the scheduled tasks solution you still aware of) is to use a glpi-agent proxy to collect inventory and send it server with a scheduled task.

As explained in [proxy-server-plugin documentation](https://glpi-agent.readthedocs.io/en/latest/plugins/proxy-server-plugin.html), you can set a specific prolog frequency in the case the proxy is not directly configured to submit inventories to GLPI (local storing case). But you'll still have to run a schedule task to inject collected inventories into glpi. Also this means you'll have another server url, even if finally you're using only one glpi server.
